src/Resources/config/config.yml

```diff
@@ -81,8 +81,10 @@ sylius_grid:
                     type: date
                     label: sylius_invoicing_plugin.ui.issued_at
                 channel:
-                    type: invoice_channel
+                    type: entity
                     label: sylius.ui.channel
+                    form_options:
+                        class: "%sylius.model.channel.class%"
```

Comment:
In the upgrade file, I see more changes. Why didn't you remove the `invoice_channel` filter?

Reply:
Fixed 👌🏻